### ISSUE TITLE
e2e flake: recovery_test.go:138: pods "test-etcd-1nids-0000" not found

### ISSUE THREAD

**Issue by hongchaodeng** [coreos/etcd-operator#627]
https://jenkins-etcd.prod.coreos.systems/job/etcd-operator/1138/console
```
--- FAIL: TestFailureRecovery (0.00s)
    --- FAIL: TestFailureRecovery/failure_recovery (0.00s)
        --- FAIL: TestFailureRecovery/failure_recovery/3_members_(all)_down (46.41s)
        	util.go:199: created etcd cluster: test-etcd-1nids
        	util.go:111: waiting size (3), etcd pods: []
        	util.go:111: waiting size (3), etcd pods: []
        	util.go:111: waiting size (3), etcd pods: [test-etcd-1nids-0000 test-etcd-1nids-0001]
        	util.go:111: waiting size (3), etcd pods: [test-etcd-1nids-0000 test-etcd-1nids-0001]
        	util.go:111: waiting size (3), etcd pods: [test-etcd-1nids-0000 test-etcd-1nids-0001 test-etcd-1nids-0002]
        	recovery_test.go:138: pods "test-etcd-1nids-0000" not found
        	util.go:214: pod (test-etcd-1nids-0003): status (Pending), cmd ([/bin/sh -c sleep 5; /usr/local/bin/etcd --data-dir=/var/etcd/data --name=test-etcd-1nids-0003 --initial-advertise-peer-urls=http://test-etcd-1nids-0003:2380 --listen-peer-urls=http://0.0.0.0:2380 --listen-client-urls=http://0.0.0.0:2379 --advertise-client-urls=http://test-etcd-1nids-0003:2379 --initial-cluster=test-etcd-1nids-0003=http://test-etcd-1nids-0003:2380 --initial-cluster-state=new --initial-cluster-token=e33c7455-aadd-4428-9461-adbc08a2f87f])
```

**Comment by hongchaodeng** [comment]
I looked into the logs again.
It seems that operator might have removed it due to disaster recovery.

**Comment by hongchaodeng** [comment]
fixed via #636
